localfs.js

```diff
@@ -11,6 +11,7 @@
  */
 
 const fs = require('fs/promises')
+const FormData = require('form-data')
```

Comment:
We shouldn't need this any more https://developer.mozilla.org/en-US/docs/Web/API/FormData

Reply:
hmm, maybe not the FormData nodejs class doesn't look to behave exactly the same as the `form-data` at least not in the tests